youtubeBot/bot.py

```diff
@@ -32,8 +32,34 @@ def send_text(self, text):
 
 
 class YoutubeBot(Bot):
-    pass  # TODO your code here!
+    def message_handler(self, message):
+        # start message in telegram with (vid 2:) to get one video
+        if str(message).startswith("vid 1:"):
+            youtube_search=message[6:]
+            video_file = download_youtube_file(youtube_search)
+            self.send_video(video_file[0])
+
+        # start message in telegram with (vid 2:) to get two video's
+        elif str(message).startswith("vid 2:"):
```

Comment:
Nice feature, how could you extend it to `vid n`, which mean send the n first video files?